app/client/components/billing/billing.tsx

```diff
@@ -100,6 +120,26 @@ const BillingRenderer = (apiResponse: MembersDataApiItem[]) => {
                     null,
                     !!productDetail.alertText
                   );
+                  const paidPlan = getMainPlan(
+                    productDetail.subscription
+                  ) as PaidSubscriptionPlan;
+                  const productInvoiceData = invoiceData
+                    .filter(
+                      invoice =>
+                        invoice.subscriptionName ===
+                        productDetail.subscription.subscriptionId
+                    )
+                    .map(invoice => {
+                      return {
+                        ...invoice,
+                        pdfPath: `/api/${invoice.pdfPath}`,
+                        currency: paidPlan.currency,
+                        currencyISO: paidPlan.currencyISO
+                      };
+                    });
```

Comment:
expression is alway cleaner...
```suggestion
                    .map(invoice => ({
                        ...invoice,
                        pdfPath: `/api/${invoice.pdfPath}`,
                        currency: paidPlan.currency,
                        currencyISO: paidPlan.currencyISO
                      })
                   );
```